udata/forms/fields.py

```diff
@@ -25,9 +25,8 @@
 
 # _ = lambda s: s
 
-RE_TAG = re.compile('^[\w \-.]*$', re.U)
-MIN_TAG_LENGTH = 2
-MAX_TAG_LENGTH = 128
+MIN_TAG_LENGTH = 3
+MAX_TAG_LENGTH = 32
```

Comment:
We need to check that all existing tags (or at least pertinent ones) are shorter than 32